mod/reportedcontent/views/default/elgg/reportedcontent.js

```diff
@@ -16,9 +16,13 @@ define(function (require) {
 			data: $form.serialize(),
 			success: function (data) {
 				if (data.status == 0) {
-					require(['elgg/lightbox'], function(lightbox) {
-						lightbox.close();
-					});
+					if ($('.elgg-form-reportedcontent-add').is('#colorbox *')) {
```

Comment:
Just as we did here: https://github.com/Elgg/Elgg/pull/9859/files/c5194f7653cf294d680e0791d78a282b42cd4f6a#diff-ffda4b57b49d28a276eee4914ecb026aR32 :smile: